src/scripts/clientUtils.js

```diff
@@ -29,20 +29,43 @@ export async function serverAddress() {
     // if (current) return Promise.resolve(current);
 
     const urls = [];
+    if (window.location.href.indexOf('/web/') !== -1) { // Do we polyfill String.prototype.includes?
+        const split = window.location.href.split('/web/');
+
+        for (let i = split.length - 1; i ; i--) {
+            urls.push(split.slice(0, i).join('/web/'));
+        }
+    }
     urls.push(window.location.origin);
     urls.push(`https://${window.location.hostname}:8920`);
     urls.push(`http://${window.location.hostname}:8096`);
```

Comment:
In my version I have removed these because I just think they are incorrect. The two last ones cause CORS issues. That's just my opinion though.

Reply:
This doesn't really matter to me either way, but I'm leaning more towards leaving as-is, and letting someone else pull them out if it is deemed necessary at a later point in time.

More opinions on this matter is welcome however.

Reply:
I'm in favor of getting this fix in and then addressing the concerns here in a follow up PR.